src/vs/workbench/browser/parts/editor/editorActions.ts

```diff
@@ -1058,18 +1059,45 @@ export class ToggleGroupSizesAction extends Action2 {
 	override async run(accessor: ServicesAccessor): Promise<void> {
 		const editorGroupService = accessor.get(IEditorGroupsService);
 
-		editorGroupService.arrangeGroups(GroupsArrangement.TOGGLE);
+		editorGroupService.toggleGroupArrangement();
 	}
 }
 
 export class MaximizeGroupAction extends Action2 {
 
 	constructor() {
 		super({
-			id: 'workbench.action.maximizeEditor',
-			title: { value: localize('maximizeEditor', "Maximize Editor Group and Hide Side Bars"), original: 'Maximize Editor Group and Hide Side Bars' },
+			id: MAXIMIZE_EDITOR_GROUP,
+			title: { value: localize('maximizeEditor', "Maximize Editor Group"), original: 'Maximize Editor Group' },
 			f1: true,
-			category: Categories.View
+			category: Categories.View,
+			precondition: ContextKeyExpr.and(MaximizedEditorGroupContext.negate(), MultipleEditorGroupsContext),
+			keybinding: {
+				weight: KeybindingWeight.WorkbenchContrib,
+				primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KeyK, KeyMod.CtrlCmd | KeyCode.KeyM),
+			}
+		});
+	}
+
+	override async run(accessor: ServicesAccessor, resourceOrContext?: URI | IEditorCommandsContext, context?: IEditorCommandsContext): Promise<void> {
+		const editorsGroupService = accessor.get(IEditorGroupsService);
+		const editorService = accessor.get(IEditorService);
+		const { group } = resolveCommandsContext(editorsGroupService, getCommandsContext(resourceOrContext, context));
+		if (editorService.activeEditor) {
```

Comment:
Why this check? This method is agnostic to editors right?

Reply:
I'm fine with removing this, however, this is how it was previously implemented with the old maximize editor group which was (or at least should have been) agnostic to editors.

Reply:
I think this is probably to prevent using the methods on empty groups but I see no reason why we couldn't support it there as well via the "..." menu.